ledger/sandbox/src/test/lib/scala/com/digitalasset/platform/store/dao/JdbcLedgerDaoSuite.scala

```diff
@@ -231,13 +248,57 @@ private[dao] trait JdbcLedgerDaoSuite extends AkkaBeforeAndAfterAll with JdbcLed
     )
   }
 
-  protected final def storeFullyTransientTransaction()(
+  /**
+    * Creates the following transaction
+    *
+    * Create A --> Exercise A
+    *              |        |
+    *              |        |
+    *              v        v
+    *           Create B  Exercise B
+    *
+    * A is visible to Charlie
+    * B is visible to Alice, Bob and Charlie
+    *
+    */
+  protected def withChildren: (Offset, LedgerEntry.Transaction) = {
```

Comment:
This doesn't seem to be used anywhere. Next PR?

Reply:
Precisely. :slightly_smiling_face: